seminars/practice_1_intro.py

```diff
@@ -32,11 +32,32 @@
 third_num = first_num / second_num
 print(f'Numbers: {first_num}, {second_num} and {third_num}')
 
-
 # Case 2: the second debugging case:
 if first_num * second_num == third_num:
     print('First number * second equals third number')
 else:
     print('First number * second does not equal third number')
 
 print('Program finished')
+
+example = 'Hello'
```

Comment:
please, do not commit these changes. Only labs

Reply:
is there a way to delete these changes?